hack/dind-cluster.sh

```diff
@@ -327,7 +327,7 @@ function nodes-are-ready() {
   # Skip the SDN node since nothing is intended to be scheduled on it.
   local node_count=$(${DOCKER_CMD} exec -t "${MASTER_NAME}" bash -c "\
 KUBECONFIG=${DEPLOYED_CONFIG_ROOT}/openshift.local.config/master/admin.kubeconfig \
-oc get nodes | grep -v ${SDN_NODE_NAME} | grep -v NotReady | grep Ready | wc -l")
+oc get nodes | grep -v SchedulingDisabled | grep -v NotReady | grep Ready | wc -l")
```

Comment:
I'd still strongly recommend `jsonpath` or `go-templates` for  extracting the values that you want to count -- this approach is fragile as the output is not stable.